hadoop-ozone/dist/src/main/license/jar-report.txt

```diff
@@ -37,23 +38,24 @@ share/ozone/lib/disruptor.jar
 share/ozone/lib/dnsjava.jar
 share/ozone/lib/error_prone_annotations.jar
 share/ozone/lib/failureaccess.jar
-share/ozone/lib/FastInfoset.jar
 share/ozone/lib/gethostname4j.jar
 share/ozone/lib/grpc-api.jar
 share/ozone/lib/grpc-context.jar
 share/ozone/lib/grpc-core.jar
 share/ozone/lib/grpc-netty.jar
-share/ozone/lib/grpc-protobuf.jar
 share/ozone/lib/grpc-protobuf-lite.jar
+share/ozone/lib/grpc-protobuf.jar
 share/ozone/lib/grpc-stub.jar
 share/ozone/lib/gson.jar
 share/ozone/lib/guava-jre.jar
 share/ozone/lib/guice-assistedinject.jar
 share/ozone/lib/guice-bridge.jar
-share/ozone/lib/guice.jar
 share/ozone/lib/guice-multibindings.jar
 share/ozone/lib/guice-servlet.jar
+share/ozone/lib/guice.jar
 share/ozone/lib/hadoop-annotations.jar
+share/ozone/lib/hadoop-annotations.jar
+share/ozone/lib/hadoop-auth.jar
 share/ozone/lib/hadoop-auth.jar
```

Comment:
Duplicate entries are present due to different jar versions for Hadoop 2 and 3.  We should not be including jars from Hadoop 2 in the Ozone distribution.